src/__tests__/standalone-needlessDisables-test.js

```diff
@@ -0,0 +1,61 @@
+import path from "path"
```

Comment:
Why do we need this file in addition to `needDisables-test.js`, which also tests the Node API?